### PR TITLE
Adds disable_pushes configuration field to guide

### PR DESCRIPTION
**Pull request by stevsmit**

### CODE REVIEW

File: modules/rn_3_13_0.adoc [hunk 1]
```diff
@@ -55,21 +55,26 @@ The following enhancements have been made to the {productname} v2 UI.
...
 The following configuration fields have been added to {productname} {producty}.
 
-[id="oauth-reassign-configuration-field"]
-=== OAuth access token reassignment configuration field 
+[id="disable-pushes-configuration-field"]
+=== Disabling pushes to the {productname} registry configuration field
+
+In some cases, a read-only option for {productname} is not possible since it requires inserting a service key and other manual configuration changes. With the release of {productname} 3.13, a new configuration field has been added: `DISABLE_PUSHES`. 
+
+When `DISABLE_PUSHES` is set to `true`, users are unable to push _new_ images to the registry; however, if the registry already contains the image and a new tag is added, the operation succeeds. All other registry operations continue as normal, for example, changing tags, editing a repository, robot account creation and deletion, user creation, and so on.
```
Comment (SeanZhao-redhat) [inline modules/rn_3_13_0.adoc:63]: Steven, pls review this chat. My understand is: "when DISABLE_PUSHES: true, can't push image even the registry already contains the image"
https://redhat-internal.slack.com/archives/G7VFPAY7Q/p1726801217423289

File: modules/rn_3_13_0.adoc [hunk 2]
```diff
@@ -55,21 +55,26 @@ The following enhancements have been made to the {productname} v2 UI.
...
+
+This field might be useful in some situations such as when {productname} administrators want to calculate their registry's quota and disable image pushing until after calculation has completed. With this method, administrators can avoid putting putting the whole registry in `read-only` mode, which affects the database, so that most operations can still be done.
```
Comment (SeanZhao-redhat) [inline modules/rn_3_13_0.adoc:66]: Can we also document some words some where for [PROJQUAY-7981](https://issues.redhat.com/browse/PROJQUAY-7981)? let user know,  tag doesn't "Permanently delete" when enable "DISABLE_PUSHES"

Comment (stevsmit) [reply modules/rn_3_13_0.adoc:66]: Done. PTAL!

File: modules/config-fields-misc.adoc [hunk 3]
```diff
@@ -94,4 +94,8 @@
   + 
 **Default:** False
 
+|*DISABLE_PUSHES* |Boolean | Disables pushes of new content to the registry while retaining all other functionality. Differs from `read-only` mode because database is not set as `read-only`. When `DISABLE_PUSHES` is set to `true`, the {productname} garbage collector is disabled. As a result, when `PERMANENTLY_DELETE_TAGS` is enabled, using the {productname} UI to permanently delete a tag does not result in the immediate deletion of a tag. Instead, the tag stays in the repository until `DISABLE_PUSHES` is set to `false`, which re-enables the garbage collector. {productname} administrators should be aware of this caveat when using `DISABLE_PUSHES` and `PERMANENTLY_DELETE_TAGS` together. +
```
Comment (SeanZhao-redhat) [inline modules/config-fields-misc.adoc:97]: "the tag stays in the repository until" better to -> "the image stays in the `storage` or (`backend storage`) until".
it does delete the tag on UI, user can't see it in the repository, it should be removed from repository but not clean from backend storage

File: modules/rn_3_13_0.adoc [hunk 4]
```diff
@@ -55,21 +55,28 @@ The following enhancements have been made to the {productname} v2 UI.
...
+
+In some cases, a read-only option for {productname} is not possible since it requires inserting a service key and other manual configuration changes. With the release of {productname} 3.13, a new configuration field has been added: `DISABLE_PUSHES`. 
+
+When `DISABLE_PUSHES` is set to `true`, users are unable to push images or image tags to the registry when using the CLI. Most other registry operations continue as normal when this feature is enabled by using the {productname} UI. For example, changing tags, editing a repository, robot account creation and deletion, user creation, and so on are all possible by using the UI. 
+
+When `DISABLE_PUSHES` is set to `true`, the {productname} garbage collector is disabled. As a result, when `PERMANENTLY_DELETE_TAGS` is enabled, using the {productname} UI to permanently delete a tag does not result in the immediate deletion of a tag. Instead, the tag stays in the repository until `DISABLE_PUSHES` is set to `false`, which re-enables the garbage collector. {productname} administrators should be aware of this caveat when using `DISABLE_PUSHES` and `PERMANENTLY_DELETE_TAGS` together.
```
Comment (SeanZhao-redhat) [inline modules/rn_3_13_0.adoc:65]: Same suggestion for "the tag stays in the repository" -> "the image stays in the storage"

File: modules/config-fields-misc.adoc (hunk 3 above)
Comment (stevsmit) [reply modules/config-fields-misc.adoc:97]: Fixed.

**Pull request merged**